hudi-common/src/main/java/org/apache/hudi/common/model/AWSDmsAvroPayload.java

```diff
@@ -45,6 +45,7 @@
 public class AWSDmsAvroPayload extends OverwriteWithLatestAvroPayload {
 
   public static final String OP_FIELD = "Op";
+  public static final String D_VALUE = "D";
```

Comment:
Use `OP_DELETE` in `isDMSDeleteRecord`?
```suggestion
  public static final String OP_DELETE = "D";
```

Reply:
Fixed this to `DELETE_OPERATION_VALUE` in https://github.com/apache/hudi/pull/13615/